PublicationsRouter/v2/oai-pmh/VERBS.md

```diff
@@ -1,15 +1,10 @@
 # OAI-PMH Verb to PubRouter API mapping
 
-This document describes the mapping that this application provides from the [OAI-PMH](http://www.openarchives.org/OAI/openarchivesprotocol.html) verbs and their associated parameters
-to the PubRouter API.
+This document describes the various [OAI-PMH](http://www.openarchives.org/OAI/openarchivesprotocol.html) verbs/requests and their interpretation by PubRouter. 
 
-There is one form for the OAI endpoint:
+The endpoint must be pointed at a specific repository like so:
 
-* For a specific repository's set of routed notifications
-
-This is mounted at a repository specific web route, of the following form:
-
-* `/repo/<repo_id>` where <repo_id> is the account identifier for the repository
+* `https://pubrouter.jisc.ac.uk/oaipmh/repo/<repo_id>` where <repo_id> is the account identifier for the repository
```

Comment:
Change above 2 lines as per earlier comments (generally avoid use of word "repository").

Change line 9:
`Note that OAI-PMH sets are not supported by PubRouter.`